src/main/java/teammates/ui/template/CourseEditInstructorPanel.java

```diff
@@ -0,0 +1,210 @@
+                                    Const.ParamsNames.INSTRUCTOR_PERMISSION_SUBMIT_SESSION_IN_SECTIONS));
+        
+        permissionInputGroup.add(createCheckBox("Sessions: View Responses and Comments", instructor,
+                                    Const.ParamsNames.INSTRUCTOR_PERMISSION_VIEW_SESSION_IN_SECTIONS));
+        
+        permissionInputGroup.add(createCheckBox(
+                                    "Sessions: Edit/Delete Responses/Comments by Others", instructor, 
+                                    Const.ParamsNames.INSTRUCTOR_PERMISSION_MODIFY_SESSION_COMMENT_IN_SECTIONS));
+        
+        return permissionInputGroup;
+    }
+
+    private List<ElementTag> createPermissionInputGroup2ForInstructorPanel(InstructorAttributes instructor) {
+        List<ElementTag> permissionInputGroup = new ArrayList<ElementTag>();
+        
+        permissionInputGroup.add(createCheckBox("View Students' Details", instructor, 
+                                    Const.ParamsNames.INSTRUCTOR_PERMISSION_VIEW_STUDENT_IN_SECTIONS));
+        
+        permissionInputGroup.add(createCheckBox("Give Comments for Students", instructor, 
+                                    Const.ParamsNames.INSTRUCTOR_PERMISSION_GIVE_COMMENT_IN_SECTIONS));
+        
+        permissionInputGroup.add(createCheckBox("View Others' Comments on Students", instructor,
+                                    Const.ParamsNames.INSTRUCTOR_PERMISSION_VIEW_COMMENT_IN_SECTIONS));
+        
+        permissionInputGroup.add(createCheckBox("Edit/Delete Others' Comments on Students", instructor,
+                                    Const.ParamsNames.INSTRUCTOR_PERMISSION_MODIFY_COMMENT_IN_SECTIONS));
+        
+        return permissionInputGroup;
+    }
+
+    private List<ElementTag> createPermissionInputGroup1ForInstructorPanel(InstructorAttributes instructor) {
+        List<ElementTag> permissionInputGroup = new ArrayList<ElementTag>();
+        
+        permissionInputGroup.add(createCheckBox("Edit/Delete Course", instructor,
+                                    Const.ParamsNames.INSTRUCTOR_PERMISSION_MODIFY_COURSE));
+        
+        permissionInputGroup.add(createCheckBox("Add/Edit/Delete Instructors", instructor,
+                                    Const.ParamsNames.INSTRUCTOR_PERMISSION_MODIFY_INSTRUCTOR));
+        
+        permissionInputGroup.add(createCheckBox("Create/Edit/Delete Sessions", instructor,
+                                    Const.ParamsNames.INSTRUCTOR_PERMISSION_MODIFY_SESSION));
+        
+        permissionInputGroup.add(createCheckBox("Enroll/Edit/Delete Students", instructor,
+                                    Const.ParamsNames.INSTRUCTOR_PERMISSION_MODIFY_STUDENT));
+        
+        return permissionInputGroup;
+    }
+    
+    private ElementTag createCheckBox(String content, InstructorAttributes instructor, String privilege) {
+        boolean isChecked = (instructor == null) || instructor.isAllowedForPrivilege(privilege);
```

Comment:
Yes, it can. It happens for the addInstructorPanel, where there is no instructor and the panel still looks like those with instructor.